project/service-wrapper.go

```diff
@@ -65,7 +66,7 @@ func (s *serviceWrapper) waitForDeps(wrappers map[string]*serviceWrapper) bool {
 	}
 
 	for _, dep := range s.service.DependentServices() {
-		if s.ignored[dep.Target] {
+		if s.ignored[dep.Target] || rUtils.IsRegionService(dep.Target) {
```

Comment:
Shall we put region links to ignored map instead? Using ignoreDep logic. Might be a bit cleaner